extractor/src/test/java/org/schabi/newpipe/extractor/services/DefaultStreamExtractorTest.java

```diff
@@ -260,13 +260,24 @@ public void testVideoStreams() throws Exception {
             assertFalse(videoStreams.isEmpty());
 
             for (final VideoStream stream : videoStreams) {
-                assertIsSecureUrl(stream.getUrl());
-                assertFalse(stream.getResolution().isEmpty());
+                if (stream.isUrl()) assertIsSecureUrl(stream.getContent());
```

Comment:
```suggestion
                if (stream.isUrl()) {
                    assertIsSecureUrl(stream.getContent());
                }
```

Reply:
We should create in a separate PR which moves all `if` statements into braces. But here I checked all the files and moved all `if` statements into braces on the parts I changed with this PR.